py/erlpack/_unpacker.pyx

```diff
@@ -305,10 +305,10 @@ cdef class ErlangTermDecoder(object):
         return self.decode_part(bytes, 0)
 
     cdef object convert_atom(self, atom):
-        if atom == 'true':
+        if atom == b'true':
             return True
-        elif atom == 'false':
+        elif atom == b'false':
             return False
-        elif atom == 'nil':
+        elif atom == b'nil':
             return None
-        return Atom(atom)
+        return Atom(atom.decode('utf-8'))
```

Comment:
if this is in a `cdef` should this be `PyUnicode_FromEncodedObject`? and also `self.encoding` instead of `utf-8`?

Reply:
done but better